exercises/webapp/.storybook/main.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  "stories": [
+    "../src/**/*.stories.mdx",
+    "../src/**/*.stories.@(js|jsx|ts|tsx)",
+    "../src/components/*.stories.@(js|jsx|ts|tsx)"
+  ],
+  "addons": [
+    "@storybook/addon-links",
+    "@storybook/addon-essentials",
+    "@storybook/addon-a11y"
```

Comment:
Did you not need controls at that point? 
```suggestion
    "@storybook/addon-a11y"
    '@storybook/addon-controls',
```

Reply:
@kipmann it's part of `addon-essentials`